### PR TITLE
Issue/1635 update tools libs

### PR DESCRIPTION
**Pull request by valllllll2000**
#### Before the PR (what is the issue or what needed to be done)
Some libs were outdated
#### The solution
Update libraries
Fix tests and code
#### Screenshots (if appropriate)
NA
#### Reviewer Checklist
* [ ] Added an explanation about the work done
* [ ] Connected the PR and the issue on Zenhub
* [ ] Added a test plan to the issue
* [ ] Updated the copyright header
* [ ] Formatted the code per our style guide
* [ ] Added a documentation (if relevant)

### CODE REVIEW

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

**Comment by valllllll2000** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- domain/src/test/java/android/text/TextUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Akvo/akvo-flow-mobile/pullRequest?prid=5744991&bid=18795423)

File: app/src/test/java/org/akvo/flow/ui/model/ViewFormMapperTest.java [hunk 1]
```diff
@@ -122,7 +122,6 @@ public void transformShouldTransformCorrectlyAllSurveyFields() {
         given(mockSurveyItem.isRegistrationForm()).willReturn(true);
         given(mockSurveyItem.getId()).willReturn("123");
         given(mockSurveyItem.getName()).willReturn("name");
-        given(mockSurveyItem.isSubmittedDataPoint()).willReturn(true);
```
Comment (tangrammer) [inline app/src/test/java/org/akvo/flow/ui/model/ViewFormMapperTest.java:125]: Why we don't expect the datapoint now?

Comment (valllllll2000) [reply app/src/test/java/org/akvo/flow/ui/model/ViewFormMapperTest.java:125]: No, that was marked as "not needed mock by JUnit"

**Pull request merged**